src/components/simulator/SimulatorStrategyType.tsx

```diff
@@ -0,0 +1,105 @@
+import { FC, useRef, KeyboardEvent } from 'react';
+import { Tooltip } from 'components/common/tooltip/Tooltip';
+import { ReactComponent as IconTwoRanges } from 'assets/icons/two-ranges.svg';
+import { ReactComponent as IconOverlappingStrategy } from 'assets/icons/overlapping-strategy.svg';
+import { cn } from 'utils/helpers';
+import { SimulatorType } from 'libs/routing/routes/sim';
+import { useNavigate } from '@tanstack/react-router';
+
+interface Props {
+  strategyType: SimulatorType;
+}
+
+export const SimulatorStrategyType: FC<Props> = ({ strategyType }) => {
+  const list = useRef<HTMLUListElement>(null);
+  const navigate = useNavigate();
+
+  const items: {
+    label: SimulatorType;
+    svg: JSX.Element;
+    tooltipText: string;
+  }[] = [
+    {
+      label: 'recurring',
+      svg: <IconTwoRanges className="h-16 w-37" />,
+      tooltipText: 'TBD',
+    },
+    {
+      label: 'overlapping',
+      svg: <IconOverlappingStrategy className="h-16 w-37" />,
+      tooltipText: 'TBD',
+    },
+  ];
+
+  const handleKeyDown = (e: KeyboardEvent) => {
+    if (e.key === 'ArrowRight' || e.key === 'ArrowLeft') {
+      const buttons = list.current?.querySelectorAll('button');
+      if (!buttons) return;
+      for (let i = 0; i < buttons.length; i++) {
+        if (buttons[i] !== document.activeElement) continue;
+        const nextIndex =
+          e.key === 'ArrowRight'
+            ? (i + 1) % buttons.length
+            : (i - 1 + buttons.length) % buttons.length;
+        return buttons[nextIndex].focus();
+      }
+    }
+  };
+
+  return (
+    <section
+      className="bg-secondary rounded-10 p-16"
+      key="simulatorTypeSelection"
+    >
+      <header className="mb-16 flex items-center justify-between">
+        <h2 className="m-0 text-18 font-weight-500">Strategy Type</h2>
+      </header>
+      <ul
+        onKeyDown={handleKeyDown}
+        ref={list}
+        role="tablist"
+        className={`grid grid-cols-2 gap-8`}
+      >
+        {items.map(({ label, svg, tooltipText }) => (
+          <li role="none" key={label} className="relative">
+            <button
```

Comment:
I would probably use a <Link> component here. Reason is, that its just a simple navigation and you benefit from stuff like right-click it and open in new tab.

Reply:
done